presto-docs/src/main/sphinx/release/release-0.223.rst

```diff
@@ -0,0 +1,26 @@
+=============
+Release 0.223
+=============
+
+General Changes
+---------------
+
+* Fix a bug where a connector may not optimize query plan even when an optimizer is specified.
```

Comment:
@highker - we should add some details around when this is triggered (if possible). Right now it doesn't convey anything clearly.

Reply:
"Fix a bug where a connector may not optimize query plan if the corresponding plugin is installed after the server is started".
Of course, there is no interface to install plugins after a server starts. So you may remove this line given there is no user impact.